### ISSUE TITLE
EQMOD: Want a small all in one board to test your code? 

### ISSUE THREAD

**Issue by MagnusThome** [EQMOD/EQM_OBDWEB#1]
@EQMOD 

Nice project! Email me at magnust@gmail.com if you want me to send you a small integrated board with all the stuff you need on one single board including buck converter with autoshutdown, SD card slot, CAN, ESP32-S3 (a RejsaCAN board)

**Comment by EQMOD** [comment]
Yes. Thank you. Will wait for the board and port the codes there.

**Comment by EQMOD** [comment]
Published a new repository for the RejsaCAN board/dongle here with the obdweb codes;

https://github.com/EQMOD/REJSACAN_OBDWEB

**Comment by black-jmyntrn** [comment]
> Published a new repository for the RejsaCAN board/dongle here with the obdweb codes;
> 
> https://github.com/EQMOD/REJSACAN_OBDWEB

I cannot get this to work. I did post an issue on this [link](https://github.com/EQMOD/REJSACAN_OBDWEB/issues/1)
